Readme.md

```diff
@@ -0,0 +1,26 @@
+Description
+===========
+
+This obs_lsstSim package provides an interface to the phosim output for the LSST Data Management software.
+
+Updating camera description
+---------------------------
+
+The camera description FITS files are built from the phosim lsst data text files. To update the camera camera description to match a new version of phosim, you will need two files from `phosim_release/data/lsst`, and the gain and saturation data from the header of a phosim generated file.
+
+1. Checkout the latest version of phosim:
+    * `git clone https://bitbucket.org/phosim/phosim_release.git`
+2. Copy these files from your phosim checkout into this repo's `description/` directory:
+    * `description/lsst/focalplanelayout.txt`
+    * `description/lsst/segmentation.txt`
+3. Generate the gain/saturation file (with obs_lsstSim setup via eups):
+    1. Build phosim (see [Using Phosim](https://bitbucket.org/phosim/phosim_release/wiki/Using%20PhoSim) for build instructions). You likely will need to create an SEDs directory (even if its empty):
+    `mkdir data/SEDs`
+    2. Run phosim to generate a simple no-background simulation (give this several hours to complete):
+    `./phosim $OBS_LSSTSIM_DIR/description/nostars_allchips -c examples/nobackground`
```

Comment:
I think this will neglect to build the wavefront sensors.  I think that's probably o.k., but we may want to note it.